CHANGELOG.md

```diff
@@ -31,6 +31,12 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
     each time the CLI is run with a profile configured this way, as it is not recommended.
 
 - The `path` positional argument for bulk `generate-template` commands is now an option (`--p/-p`).
+- Below `search` subcommands accept argument `--format/-f` to display result in formats `csv`, `table`, `json`, `raw-json`:
+    - BREAKING CHANGE: Default output format is changed to `table` format from `raw-json`, returns a paginated response.
```

Comment:
If it is under `changed` header in the Changelog, you don't need to declare `BREAKING CHANGE` here